### PR TITLE
[SCB-1513]merge config items according to priority.

### PR DESCRIPTION
**Pull request by zhangchi02**

### CODE REVIEW

File: dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ParseConfigUtils.java [hunk 1]
```diff
@@ -130,11 +134,23 @@ private void doRefreshItems() {
   }
 
   private Map<String, Object> mergeDimensionItems(Map<String, Map<String, Object>> items) {
-    Map<String, Object> flatMap = new HashMap<>();
-    return items.values().stream().reduce(flatMap, (result, item) -> {
-      result.putAll(item);
-      return result;
-    });
+	Map<String, Object> flatMap = new HashMap<>();
```
Comment (jeho0815) [inline dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ParseConfigUtils.java]: Your code format style is wrong, please use the format file under https://github.com/apache/servicecomb-java-chassis/tree/master/etc

File: dynamic-config/config-cc/src/test/java/org/apache/servicecomb/config/client/TestParseConfigUtils.java [hunk 2]
```diff
@@ -80,4 +81,26 @@ public void testNotifyItemsChangedNeedRefresh() {
...
+	items.put("application", application);
+	items.put("service@app", service);
+	items.put("service@app#version", version);
+	items.put("service@app#version!tag", tag);
+    try {
+      Deencapsulation.invoke(pc, "mergeDimensionItems", items);
+    } catch (Exception e) {
+      status = false;
+    }
+    Assert.assertTrue(status);
```
Comment (jeho0815) [inline dynamic-config/config-cc/src/test/java/org/apache/servicecomb/config/client/TestParseConfigUtils.java]: the dimentions should be out of order and  you can add some key/values and , then check the merged result

**Comment by liubao68** [comment]
Can you please resolve the conflicsts ?

**Comment by liubao68** [comment]
Please retrigger CI after #1340 is merged or some test case will randomly fail.

**Pull request merged**